libs/providers/flagd/src/lib/flagd-provider.ts

```diff
@@ -43,7 +44,10 @@ export class FlagdProvider implements Provider {
     private readonly logger?: Logger,
     service?: Service,
   ) {
-    this._service = service ? service : new GRPCService(getConfig(options), undefined, logger);
+    const config = getConfig(options);
+
+    this._service = service ? service :
+      (config.resolverType === 'rpc' ? new GRPCService(config, undefined, logger) : new InProcessService(config));
```

Comment:
```suggestion
      (config.resolverType === 'in-process' ? new InProcessService(config) : new GRPCService(config, undefined, logger));
```

Since `rpc` is supposed to be the default, it should be used if someone passes in an invalid resolver type.

Reply:
This is done :)